### ISSUE TITLE
No dark tray icon

### ISSUE THREAD

**Issue by Schattenjager-RC** [FreeRDP/Remmina#905]
I am using Ubuntu's Radiance light theme, but Remmina's tray icon is always light too. It merges with the background and difficult to see.
![unity-panel_001](https://cloud.githubusercontent.com/assets/6198836/16169430/c20cdf8c-353e-11e6-8f90-35ded587c357.png)
Please add dark inverted icon and the ability to automatically select dark or light icon depending on the system theme. Or maybe just the ability to choose icon in settings - some variants like colour/light/dark. 
Thanks in advance.

Here is some information about my system:
* Client: Ubuntu 16.04 LTS
* Remmina version: 1.2.0-rcgit.14
* Desktop environment: Unity

**Comment by antenore** [comment]
Thanks for the report. 

@wa4557 would you mind providing a dark icon? I'll work on the rest. Thanks in advance.

**Comment by Schattenjager-RC** [comment]
@antenore
I've just inverted the 24x24 icon with XnViewMP.
You can use it)

**Comment by antenore** [comment]
Thanks :-) 
Let me investigate  which options we have to switch between icons.

**Comment by Schattenjager-RC** [comment]
You are welcome)

**Comment by Schattenjager-RC** [comment]
Maybe you should look how it's implemented in applications like Shutter or Redshift.

**Comment by antenore** [comment]
Good idea, in the meanwhile I've found this

http://lzone.de/blog/Detecting%20a%20Dark%20Theme%20in%20GTK

**Comment by Schattenjager-RC** [comment]
It looks Interesting) A lot of applications automatically change icon colour, even very simple indicators can do it. Not sure if it's the same implementation.

**Comment by Schattenjager-RC** [comment]
But I think it may be much easier to add the ability to select the icon in the settings.

**Comment by antenore** [comment]
I've wrote down some few lines of code... It's not perfect but it should work.

**Comment by antenore** [comment]
@Schattenjager-RC If you want we add your name in the Remmina ABout Dialog (and in the AUTHORS file) contact me by email (my email is in the about dialog as well)

**Comment by Chugajstyr** [comment]
Possibly same trouble with tray icon (Mint 17.3, MATE Desktop, standart Mint-X desktop theme, Remmina 1.2.0-rcgit.14)
![remmina screenshot at 2016-06-20 20 21 37](https://cloud.githubusercontent.com/assets/16296772/16204024/a3f84e08-3725-11e6-86b0-613d7cd75112.png)
And menu button strange looks:
![remmina screenshot at 2016-06-20 20 22 41](https://cloud.githubusercontent.com/assets/16296772/16204064/c758ebfa-3725-11e6-9d39-cf5903fde02e.png)

**Comment by antenore** [comment]
See #902 about menu button.

The panel icon fix will be merged as soon as possible

**Comment by box293** [comment]
I have the same issue with Mint MATE just as @Chugajstyr reported.

**Comment by Schattenjager-RC** [comment]
@antenore
Great, thanks a lot.
Regarding your proposal - for now I prefer to stay just anonymous "contributor")

**Comment by box293** [comment]
This fixed it for me on Linux Mint 17.3

```
wget -O /usr/share/icons//hicolor/16x16/apps/remmina-panel.png https://raw.githubusercontent.com/FreeRDP/Remmina/next/remmina/desktop/16x16/apps/remmina-panel-inverted.png
wget -O /usr/share/icons//hicolor/22x22/apps/remmina-panel.png https://raw.githubusercontent.com/FreeRDP/Remmina/next/remmina/desktop/22x22/apps/remmina-panel-inverted.png
wget -O /usr/share/icons//hicolor/24x24/apps/remmina-panel.png https://raw.githubusercontent.com/FreeRDP/Remmina/next/remmina/desktop/24x24/apps/remmina-panel-inverted.png
gtk-update-icon-cache /usr/share/icons/hicolor/
```
